.betterer.results

```diff
@@ -800,7 +800,8 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "3"]
+      [0, 0, 0, "Unexpected any. Specify a different type.", "3"],
+      [0, 0, 0, "Do not use any type assertions.", "4"]
```

Comment:
It was probably added because I am reusing `EchoEvent` which has `any` in its implementation. I don't think that this should be an "issue" in this particular case, as the any should be fixed at the `EchoEvent` side?

Reply:
this is because of this line: `event as Omit<EchoEvent, 'meta'>` (it doesn't like the assertion `as`.

Reply:
I have added a couple of suggestions that removes the need of adding this to the betterer.results.